x11-misc/xsensors/xsensors-0.70-r1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit autotools eutils
+
+DESCRIPTION="A hardware health information viewer, interface to lm-sensors"
+HOMEPAGE="http://www.linuxhardware.org/xsensors/"
+SRC_URI="http://www.linuxhardware.org/xsensors/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE=""
+
+RDEPEND="x11-libs/gtk+:2
+	>=sys-apps/lm_sensors-3"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+src_prepare() {
+	epatch "${FILESDIR}"/${P}-gtk220.patch
+	eapply_user
+
+	sed -i \
+		-e '/-DG.*_DISABLE_DEPRECATED/d' \
+		-e 's:-Werror:-Wall:' \
+		src/Makefile.am configure.in || die
+
+	mv configure.in configure.ac || die #426262
+
+	eautoreconf
+}
+
+src_install() {
+	emake DESTDIR="${D}" install || die
+	dodoc AUTHORS ChangeLog README TODO
+}
```

Comment:
nix this entire `src_install`, all of it should be in `default_src_install` of EAPI 6